### DISCUSSION TITLE
请问能增加一个sleep()的功能吗

### DISCUSSION THREAD

**Discussion by kafeiya** [shuax/MouseInc.Settings#137] [Ideas]
比如我想打开TIm并登录，需要三步操作
1. "Execute"打开tim
2. "MouseMove"鼠标移动到登录按钮位置
3. ["SendClick", "left"] 鼠标点击，
可是执行1之后，程序还没加载完，第2，3步就已经运行完了

如果增加sleep()功能的话，就能实现类似脚本的功能，

**Comment by shuax** [comment]
有delay，就是你说的sleep

**Reply by kafeiya** [reply]
感谢，原来手册里有这个功能，是我之前没有仔细看了。
